x-pack/plugins/case/server/client/comments/add_comment.test.ts

```diff
@@ -0,0 +1,212 @@
+/*
```

Comment:
nit -> the file name can just be `add.test.ts` since we are already in the folder of comments